content/open-source/sigstore/policy-controller/warn-deprecated-k8s-registry.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Notification of Kubernetes Registry Deprecation"
```

Comment:
This one is not finished yet but was planning on having a sister one for the policy-controller alongside the one for Enforce